doc/source/dev/contributor/continuous_integration.rst

```diff
@@ -44,8 +45,8 @@ Azure
 -----
 * ``Lint``: PEP8 and code style
 * ``Windows Python``: test suite runs for Windows
-* ``Linux_Python_37_32bit_full``
-* ``wheel_optimized_gcc48``
+* ``Linux_Python_xx_32bit_full``: 32-bit Linux
+* ``wheel_optimized_gcc``
```

Comment:
Could you add a description here like there is for the others?

Reply:
Ok I will update this. For the other jobs I did not do much more since we will shuffle this with Meson. When I added this file originally, we said we did not want to go too deep because things tend to change often and we don't want to bother updating this all the time.

Reply:
Then to keep things consistent, I would suggest just omitting the names of jobs that are not important enough to define, and explain that only some of the important jobs are listed.

Reply:
With the update we would have all jobs I think. (Aside from defining all different jobs per OS).